edx_notifications/stores/sql/migrations/0007_auto__add_sqlusernotificationarchive__add_unique_sqlusernotificationar.py

```diff
@@ -1,114 +0,0 @@
-            'renderer_context': ('django.db.models.fields.TextField', [], {'null': 'True'})
-        },
-        'edx_notifications.sqlusernotification': {
-            'Meta': {'ordering': "['-created']", 'unique_together': "(('user_id', 'msg'),)", 'object_name': 'SQLUserNotification', 'db_table': "'edx_notifications_usernotification'"},
-            'created': ('model_utils.fields.AutoCreatedField', [], {'default': 'datetime.datetime.now'}),
-            'id': ('django.db.models.fields.AutoField', [], {'primary_key': 'True'}),
-            'modified': ('model_utils.fields.AutoLastModifiedField', [], {'default': 'datetime.datetime.now'}),
-            'msg': ('django.db.models.fields.related.ForeignKey', [], {'to': "orm['edx_notifications.SQLNotificationMessage']"}),
-            'read_at': ('django.db.models.fields.DateTimeField', [], {'null': 'True', 'db_index': 'True'}),
-            'user_context': ('django.db.models.fields.TextField', [], {'null': 'True'}),
-            'user_id': ('django.db.models.fields.IntegerField', [], {'db_index': 'True'})
-        },
-        'edx_notifications.sqlusernotificationarchive': {
-            'Meta': {'ordering': "['-created']", 'unique_together': "(('user_id', 'msg'),)", 'object_name': 'SQLUserNotificationArchive', 'db_table': "'edx_notifications_usernotificationarchive'"},
-            'created': ('model_utils.fields.AutoCreatedField', [], {'default': 'datetime.datetime.now'}),
-            'id': ('django.db.models.fields.AutoField', [], {'primary_key': 'True'}),
-            'modified': ('model_utils.fields.AutoLastModifiedField', [], {'default': 'datetime.datetime.now'}),
-            'msg': ('django.db.models.fields.related.ForeignKey', [], {'to': "orm['edx_notifications.SQLNotificationMessage']"}),
-            'read_at': ('django.db.models.fields.DateTimeField', [], {'null': 'True', 'db_index': 'True'}),
-            'user_context': ('django.db.models.fields.TextField', [], {'null': 'True'}),
-            'user_id': ('django.db.models.fields.IntegerField', [], {'db_index': 'True'})
-        },
-        'edx_notifications.sqlusernotificationpreferences': {
-            'Meta': {'object_name': 'SQLUserNotificationPreferences', 'db_table': "'edx_notifications_usernotificationpreferences'"},
-            'created': ('model_utils.fields.AutoCreatedField', [], {'default': 'datetime.datetime.now'}),
-            'id': ('django.db.models.fields.AutoField', [], {'primary_key': 'True'}),
-            'modified': ('model_utils.fields.AutoLastModifiedField', [], {'default': 'datetime.datetime.now'}),
-            'preference': ('django.db.models.fields.related.ForeignKey', [], {'to': "orm['edx_notifications.SQLNotificationPreference']"}),
-            'user_id': ('django.db.models.fields.IntegerField', [], {'db_index': 'True'}),
-            'value': ('django.db.models.fields.CharField', [], {'max_length': '255'})
-        }
-    }
-
-    complete_apps = ['edx_notifications']
```

Comment:
Have all of these migrations been rolled into the initial migration?  Does this have any implications for rollback strategy?